src/api/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-6.2.1-SNAPSHOT
+6.3.0-SNAPSHOT
```

Comment:
@felipetavares reminder to regenerate the secondary flow to update this after we merge